tests/e2e/test_e2e.py

```diff
@@ -0,0 +1,29 @@
+import random
+
+import pytest
+
+from mpt_api_client import MPTClient
+from mpt_api_client.exceptions import MPTAPIError
+
+
+@pytest.mark.flaky(reruns=5, reruns_delay=0.01)  # noqa: WPS432
+@pytest.mark.e2e
```

Comment:
Could you remind me what marks should be here? Looks a little bit extra to have e2e folder and e2e mark at the same time